### ISSUE TITLE
ContestSnakeYAMLLoaderTest JUnit "testCCS1Load" is failing

### ISSUE THREAD

**Issue by clevengr** [pc2ccs/pc2v9#63]
**Describe the issue**:

The JUnit test _**ContestSnakeYAMLLoaderTest.testCCS1Load()**_ fails when run on GitLab (as a result of an update to the pc2ccs/pc2v9 develop branch).  However, running this test _succeeds_ on at least two different local machines:  my Win10 Desktop and my Win8.1 laptop.

There is apparently something in this JUnit that operates differently on GitLab than it does under Eclipse; this difference needs to be found and mitigated.

**To Reproduce**: 

Push a trivial change to the pc2ccs/pc2v9 (that is, push to either the _develop_ branch or the _master_ branch on the authoritative PC2v9 repo).  For example, add an additional blank line to an existing file; anything that should not cause a build to fail.

**Expected behavior**: 

The build on GitLab succeeds and posts an updated build to the ECS "AllGitLabBuilds" page.
  
**Actual behavior**: 

The GitLab build fails, specifically because the JUnit test _**ContestSnakeYAMLLoaderTest.testCCS1Load()**_ fails.

**Environment**: 
Local machine: Win10 with jre 1.8.0_201 and Win8.1 with the same jre both succeed (all Junits pass).

Remote system:  doesn't matter; any push to pc2ccs/pc2v9 seems to cause this issue.

**Log Info**:   See attached log from GitLab: [JUnitBuildFailure.18March2020.txt](https://github.com/pc2ccs/pc2v9/files/4357604/JUnitBuildFailure.18March2020.txt)

**Screenshots**:  N/A

**Additional context**: 

According to the attached log, GitLab is using OpenJDK 11, whereas my local machine is using jre 1.8.0_201; perhaps this has something to do with the issue?    (If we are expecting to be  able to be able to run with Java 8, shouldn't we be BUILDING with Java 8?)

**Comment by lane55** [comment]
October 2020 - what is the current state of the master build?  Is it still failing?

**Comment by clevengr** [comment]
I would assume yes, the master branch build is "still failing".  However,
it doesn't "fail" until/unless it gets "built", and it doesn't get "built"
until something is pushed to it.  As I explained in my previous note, I
think I have correctly documented what needs to get "pushed" to the master
to verify that it (no longer) fails.  I also pointed out that we can either
do that with a cherry pick, or we can wait until we merge all the stuff on
"develop" into "master"; either of these should allow the master branch to
build properly.  But either way, we won't know for certain until we push
the fix to it (and trigger a rebuild).

john
